src/olm/session/mod.rs

```diff
@@ -662,7 +662,7 @@ mod test {
 
         let mut messages: Vec<messages::OlmMessage> = Vec::new();
         for i in 0..(MAX_MESSAGE_KEYS + 2) {
-            messages.push(bob_session.encrypt(format!("Message {}", i).as_str()).into());
+            messages.push(bob_session.encrypt(format!("Message {i}").as_str()).into());
```

Comment:
This one definitely looks unrelated.

For absolute clarity, I'd suggest making this a separate commit.

Reply:
But it is a separate commit?